client/src/components/MapElement/MapElement.js

```diff
@@ -96,7 +23,7 @@ export function MapElement({ className }) {
         </h1>
       </span>
     )
-  }, [appView, activeTripId, newTripForm])
+  }, [appView, activeTripId, NewTripForm])
```

Comment:
huh its curious that we have to watch an import. but if thats needed then it's okay